fuel/streams.py

```diff
@@ -150,6 +158,11 @@ class ServerDataStream(AbstractDataStream):
 
     Parameters
     ----------
+    sources : tuple of strings
+        The names of the data sources returned by this data stream.
+    produces_examples : bool
+        Whether this data stream produces examples (as opposed to batches
```

Comment:
Ideally this should eventually be auto-negotiated. Please remember to file a ticket.